test/library/standard/BigInteger/apiTest.chpl

```diff
@@ -0,0 +1,204 @@
+use BigInteger;
+use Reflection;
+
+const a = new bigint("123456789012345678901234567890");
+const b = new bigint("123456789012345678901234567891");
+const c = new bigint("246913578024691357802469135780");
+const d = new bigint("-246913578024691357802469135780");
+
+proc writeIfFalse(input: bool, lineNo) {
+  if !input then
+    writeln("Failed on ", lineNo:string);
+}
+
+writeIfFalse((a+1) == b, getLineNumber());
```

Comment:
Why `writeIfFalse(test, getLineNumber());` instead of `assert(test);`?

Reply:
Alright, you got me this time... Fixed.